com.unity.formats.alembic/Documentation~/time_AlembicShot.md

```diff
@@ -8,14 +8,12 @@ To create an Alembic Shot:
 
 1. Drag a GameObject with an [Alembic Stream Player](ref_StreamPlayer.md) component onto the Clips view in Timeline.
 
-   If Timeline is empty, create a temporary track so that you can see the Clips view.
+   If Timeline is empty, create an Alembic track so that you can see the Clips view.
```

Comment:
I think we can remove this line completely. We don't need to create an empty track for this to work, we can just drag it into the empty space of the timeline. I'm not quite sure why it's written this way but I suspect it was an ealier and more limited version of Timeline.
![abcclip_001](https://github.com/Unity-Technologies/com.unity.formats.alembic/assets/3811928/b8e48954-c258-4bf2-bcc7-e807293ea09c)

Reply:
Removed in [39bc8b8](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/533/commits/39bc8b88ab949acae1756742f10cdc02ffd35674)